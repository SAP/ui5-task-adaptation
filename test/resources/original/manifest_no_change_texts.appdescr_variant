{
    "fileName": "manifest",
    "layer": "CUSTOMER_BASE",
    "fileType": "appdescr_variant",
    "reference": "sap.ui.rta.test.variantManagement.business.service",
    "id": "customer.sap.ui.rta.test.variantManagement.business.service",
    "namespace": "apps/sap.ui.rta.test.variantManagement.business.service/appVariants/customer.sap.ui.rta.test.variantManagement.business.service/",
    "content": [
        {
            "changeType": "appdescr_ui5_addNewModelEnhanceWith",
            "content": {
                "modelId": "i18n|sap.suite.ui.generic.template.ListReport|SEPMRA_C_PD_Product"
            }
        }
    ]
}