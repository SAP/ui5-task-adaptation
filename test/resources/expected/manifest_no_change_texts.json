{
    "_version": "1.8.0",
    "sap.app": {
        "ach": "CA-UI5-FL-RTA",
        "applicationVersion": {
            "version": "1.0.0"
        },
        "description": "{{appDescription}}",
        "i18n": "i18n/i18n.properties",
        "id": "customer.sap.ui.rta.test.variantManagement.business.service",
        "title": "{{appTitle}}",
        "type": "application"
    },
    "sap.ui5": {
        "appVariantIdHierarchy": [{
            "appVariantId": "sap.ui.rta.test.variantManagement.business.service",
            "version": "1.0.0"
        }],
        "dependencies": {
            "libs": {
                "sap.m": {},
                "sap.ui.core": {},
                "sap.ui.dt": {},
                "sap.ui.rta": {}
            },
            "minUI5Version": "1.60.2"
        },
        "models": {
            "i18n": {
                "type": "sap.ui.model.resource.ResourceModel",
                "uri": "i18n/i18n.properties"
            },
            "i18n|sap.suite.ui.generic.template.ListReport|SEPMRA_C_PD_Product": {
                "settings": {
                    "enhanceWith": [
                        {
                            "bundleName": "customer.sap.ui.rta.test.variantManagement.business.service.appvariant-a6eed165.i18n.i18n"
                        }
                    ]
                },
                "type": "sap.ui.model.resource.ResourceModel",
                "uri": "i18n/ListReport/SEPMRA_C_PD_Product/i18n.properties"
            },
            "i18n|sap.suite.ui.generic.template.ObjectPage|SEPMRA_C_PD_Product": {
                "type": "sap.ui.model.resource.ResourceModel",
                "uri": "i18n/ObjectPage/SEPMRA_C_PD_Product/i18n.properties"
            }
        },
        "rootView": {
            "async": true,
            "id": "app",
            "type": "XML",
            "viewName": "customer.sap.ui.rta.test.variantManagement.business.service.view.Root"
        },
        "routing": {
            "config": {
                "async": true,
                "bypassed": {
                    "target": [
                        "main"
                    ]
                },
                "controlAggregation": "pages",
                "controlId": "app",
                "routerClass": "sap.m.routing.Router",
                "viewPath": "customer.sap.ui.rta.test.variantManagement.business.service.view",
                "viewType": "XML"
            }
        }
    }
}